python/py-editdistance/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        aflc editdistance 0.5.3 v
+revision            0
+name                py-${github.project}
+categories-append   textproc
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Fast implementation of the edit distance (Levenshtein distance).
+
+long_description    ${description} This library simply implements\
+                    Levenshtein distance with C++ and Cython. The\
+                    algorithm used in this library is proposed by\
+                    Heikki Hyyrö, \"Explaining and extending the\
+                    bit-parallel approximate string matching algorithm\
+                    of Myers\", (2001).
+
+checksums           rmd160  c2a9bbae79e29d38e1cc3cfb5b7449ab4b6275dc \
+                    sha256  7fde6523516bffd10f0a7913134562111d7b0f445ace5a3010160cead7c2bf57 \
+                    size    27988
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE \
+            ${destroot}${docdir}
+    }
+
+    test.run        yes
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
+    test.cmd        python${python.branch}
+    test.target     test/test_${github.project}.py
+
```

Comment:
the test phase cannot have worked... as the environment is not setup correctly when things get compiled. You should instead add the following before ```test.run```:
```
    pre-test {
        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
    }
```

additionally, please: use  ```${python.bin}``` for the ```test.cmd```

Reply:
Thanks done. 🦃🦃🦃